### PR TITLE
Avoid NullPointerException thrown when some parameter omitted

### PR DESCRIPTION
**Pull request by yoichiro**
# Abstract

This pull request has a diff which avoids `NullPointerException` is thrown when some parameter is omitted on an `azureWebapp` definition in a `build.gradle` file. Users can know the reason clearly by seeing the concrete error message.

# Behavior that changed

In the latest `master` branch code, if a `authentication` property is omitted, you will see the following output:

```bash
$ gradle azureWebappDeploy

> Task :azureWebappDeploy FAILED
Start deploying to Web App yoichirowa1...
Target Web App doesn't exist. Creating a new one...
com.microsoft.azure.gradle.webapp.handlers.WindowsRuntimeHandlerImpl

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':azureWebappDeploy'.
> java.lang.NullPointerException (no error message)
...
```

After applying this pull request, you will see the following output against same situation:

```bash
$ gradle azureWebappDeploy

> Task :azureWebappDeploy FAILED
Start deploying to Web App yoichirowa1...
Target Web App doesn't exist. Creating a new one...
com.microsoft.azure.gradle.webapp.handlers.WindowsRuntimeHandlerImpl

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':azureWebappDeploy'.
> 'authentication' is not configured in build.gradle.
...
```

# Target properties

In this pull request, the situation at omitted the following properties are supported:

* `azureWebapp.appName`
* `azureWebapp.authentication`
* `azureWebapp.authentication.type`
* `azureWebapp.deployment`
* `azureWebapp.deployment.type`
* `azureWebapp.appService.javaVersion`

# How to check them and output an error message

Basically, to check whether a parameter is omitted or not,

* Check whether the returned value of the target property is `null` or not before the line where accessing to the target property.
* If the returned value is `null`, throw a `GradleException` with an error message including the property name.

### CODE REVIEW

**Comment by yoichiro** [comment]
@lenala Could you review this pull request?
